app/controllers/api/v1/accounts/base_controller.rb

```diff
@@ -14,7 +14,7 @@ def ensure_current_account
     account = Account.find(params[:account_id])
     if current_user
       account_accessible_for_user?(account)
-    elsif @resource&.is_a?(AgentBot)
```

Comment:
Was this an intended change?

Reply:
Yep, the safe operator here seemed like we are suppressing the chances of an error.